hack/ci/run-cilium-e2e-test.sh

```diff
@@ -30,7 +30,7 @@ function cleanup() {
 trap cleanup EXIT SIGINT SIGTERM
 
 export KIND_CLUSTER_NAME="${SEED_NAME:-kubermatic}"
-export KUBERMATIC_YAML=hack/ci/testdata/kubermatic_cilium.yaml
+export KUBERMATIC_YAML=hack/ci/testdata/kubermatic_headless.yaml
```

Comment:
`hubble` addon is now installed explicitly, so no need to configure KKP to have it as a default addon. This also makes it easier to test on dev, where hubble might not be setup as a default addon.